package.json

```diff
@@ -19,8 +19,6 @@
     "build": "yarn workspaces foreach --all --topological-dev --parallel run build",
     "test:unit": "yarn workspaces foreach --all run test:unit",
     "dev:server": "node test-server/server",
-    "dev:storybook": "cd test-storybook && yarn storybook",
-    "start": "yarn build:watch",
     "lint": "yarn workspaces foreach --all --parallel run lint",
     "prettier": "prettier",
```

Comment:
I'm not seeing any reference to `yarn run prettier` (in the GitHub workflow files) or `yarn prettier` (in any of the package.json scripts); @tmeasday can we remove that command (asking you since you put the `prettier` command in here initially)?

Reply:
Fine by me

Reply:
Actually, I see we are using prettier from the `lint-staged` command, which itself is [run](https://github.com/chromaui/chromatic-e2e/blob/main/.husky/pre-commit#L4) by `husky`. I'll leave it in!